pkg/pillar/zedcloud/send.go

```diff
@@ -142,7 +140,7 @@ func VerifyAllIntf(ctx ZedCloudContext,
 				break
 			}
 			resp, _, rtf, err := SendOnIntf(ctx, url, intf, 0, nil, allowProxy)
-			if rtf {
+			if rtf == types.ExtStatusRemTempFail {
```

Comment:
Shouldn't this Verify function also return all the enum values and the caller handle the Miss case by requesting the certs?

Reply:
I currently make the zedagent is the only guy who will have the need for certs chain and verify the payload. in this VerifyAllIntf part, it only sends up to cloud, not receiving stuff back.

Reply:
My concern is whether Verify will fail if it doesn't get the updated certificates. And it doesn't re-read the server0 cert file either. Hence nim wouldn't react when we rekey the server. For next sprint.